extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/dataportabilityproject/transfer/smugmug/photos/SmugMugPhotosImporter.java

```diff
@@ -119,33 +141,32 @@ void importSingleAlbum(UUID jobId, String folder, PhotoAlbum inputAlbum) throws
   void importSinglePhoto(UUID jobId, PhotoModel inputPhoto) throws IOException {
     // Set up photo
     InputStreamContent content =
-            new InputStreamContent(null, getImageAsStream(inputPhoto.getFetchableUrl()));
+        new InputStreamContent(null, getImageAsStream(inputPhoto.getFetchableUrl()));
 
     // Find album to upload photo to
-    String newAlbumKey = jobStore.findData(TempPhotosData.class, jobId).lookupNewAlbumId
-            (inputPhoto.getAlbumId());
+    String newAlbumKey =
+        jobStore.findData(TempPhotosData.class, jobId).lookupNewAlbumId(inputPhoto.getAlbumId());
     checkState(
-            !Strings.isNullOrEmpty(newAlbumKey), "Cached album key for %s is null", inputPhoto
-                    .getAlbumId());
+        !Strings.isNullOrEmpty(newAlbumKey),
+        "Cached album key for %s is null",
+        inputPhoto.getAlbumId());
 
     // Upload photo
     smugMugInterface.postRequest(
-            "http://upload.smugmug.com/",
-            content,
-            // Headers from: https://api.smugmug.com/api/v2/doc/reference/upload.html
-            ImmutableMap.of(
-                    "X-Smug-AlbumUri", "/api/v2/album/" + newAlbumKey,
-                    "X-Smug-ResponseType", "json",
-                    "X-Smug-Version", "v2"),
-            new TypeReference<ImageUploadResponse>() {});
-
+        "http://upload.smugmug.com/",
+        content,
+        // Headers from: https://api.smugmug.com/api/v2/doc/reference/upload.html
+        ImmutableMap.of(
```

Comment:
Maybe this map creation can be broken up into a different method that takes in the album key

Reply:
Yea, I was planning on moving this with more substantial changes when switching the postRequest function to using the scribe API and signing in a followup PR. I didn't actually modify the Importer yet aside from the type changes necessary for compiling.